hazelcast/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -984,13 +985,33 @@ public ClientConfig setInstanceTrackingConfig(@Nonnull InstanceTrackingConfig in
         return this;
     }
 
+    /**
+     * Returns the configuration for the SQL feature.
+     * @since 5.2
+     */
+    @Nonnull
+    public ClientSqlConfig getSqlConfig() {
+        return sqlConfig;
+    }
+
+    /**
+     * Sets the configuration for the SQL feature.
+     * @since 5.2
+     */
+    @Nonnull
+    public ClientConfig setSqlConfig(ClientSqlConfig sqlConfig) {
+        Preconditions.checkNotNull(sqlConfig, "sqlConfig");
```

Comment:
nit: I'd add "provided sqlConfig cannot be null" instead of just "sqlConfig". Always a little bit of context

Reply:
Actually there is better method for that:
```java
    public static <E> E isNotNull(E argument, String argName) {
        if (argument == null) {
            throw new IllegalArgumentException(format("argument '%s' can't be null", argName));
        }

        return argument;
    }
```
I will fix all of usages of ```Preconditions.checkNotNull``` in that class